packages/devtools_app/lib/src/screens/deep_link_validation/deep_link_list_view.dart

```diff
@@ -72,266 +79,136 @@ class _DeepLinkListViewMainPanel extends StatelessWidget {
   });
   final List<LinkData> linkDatas;
-  final TableViewType tableView;
+  final TableViewType viewType;
   final DeepLinksController controller;
 
   @override
   Widget build(BuildContext context) {
-    final ColumnData<LinkData> domain = DomainColumn();
-    final ColumnData<LinkData> path = PathColumn();
-
-    return FlatTable(
-      keyFactory: (node) => ValueKey(node.toString),
-      data: linkDatas,
-      dataKey: 'deep-links',
-      autoScrollContent: true,
-      columns: <ColumnData>[
-        if (tableView != TableViewType.pathView) domain,
-        if (tableView != TableViewType.domainView) path,
-        SchemeColumn(),
-        OSColumn(),
-        if (!controller.showSpitScreen) ...[
-          StatusColumn(),
-          NavigationColumn(),
-        ],
-      ],
-      selectionNotifier: controller.selectedLink,
-      defaultSortColumn: tableView == TableViewType.pathView ? path : domain,
-      defaultSortDirection: SortDirection.ascending,
-      onItemSelected: (item) => controller.showSpitScreenNotifier.value = true,
-    );
-  }
-}
-
-class _ValidationDetailScreen extends StatelessWidget {
-  const _ValidationDetailScreen({
-    required this.linkData,
-    required this.tableView,
-    required this.controller,
-  });
+    final ColumnData<LinkData> domain = DomainColumn(controller);
+    final ColumnData<LinkData> path = PathColumn(controller);
 
-  final LinkData linkData;
-  final TableViewType tableView;
-  final DeepLinksController controller;
-
-  @override
-  Widget build(BuildContext context) {
     return Padding(
-      padding: const EdgeInsets.symmetric(horizontal: largeSpacing),
-      child: Column(
-        crossAxisAlignment: CrossAxisAlignment.start,
-        children: [
-          Row(
-            mainAxisAlignment: MainAxisAlignment.spaceBetween,
-            children: [
-              const Text('Selected deep link validation details'),
-              IconButton(
-                onPressed: () =>
-                    controller.showSpitScreenNotifier.value = false,
-                icon: const Icon(Icons.close),
-              ),
-            ],
-          ),
-          Text(
-            'This tool helps you diagnose Universal Links, App Links,'
-            ' and Custom Schemes in your app. Web checks are done for the web association'
-            ' files on your website. App checks are done for the intent filters in'
-            ' the manifest and info.plist file, routing issues, URL format, etc.',
-            style: Theme.of(context).textTheme.bodySmall,
-          ),
-          const Text('Domain check'),
-          Expanded(child: _DomainCheckTable(linkData: linkData)),
+      padding: const EdgeInsets.only(top: denseSpacing),
+      child: FlatTable(
+        keyFactory: (node) => ValueKey(node.toString),
+        data: linkDatas,
+        dataKey: 'deep-links',
+        autoScrollContent: true,
+        headerColor: Theme.of(context).colorScheme.deeplinkTableHeaderColor,
```

Comment:
we should make the styling for tables in this screen consistent with the other tables across DevTools (CC @nancyhu2023)

Reply:
talked offline to Nancy. 
According to the new color upgrade all table header color should be black.
https://www.figma.com/file/jX0mCLmruKGdnqIaIX1t5Z/Flutter-DevTools-Colors-Upgrade?type=design&node-id=53634-134804&mode=design&t=gcZucLNl7ZgKeNZT-0

Reply:
I can file a separated to change all tables header color in dev tool to black, what do you think?